### ISSUE TITLE
Distribution of Jupyter Notebooks and the Example Datasets

### ISSUE THREAD

**Issue by gspeed0689** [UtrechtUniversity/gis-python-power#1]
How do we want to distribute the Jupyter Notebooks and the example datasets for use during the workshops? 

The current example datasets in the folder are many megabytes or gigabytes, however they will be reduced to smaller sizes to ease processing and load. 

- [Binder](https://mybinder.org/) could be a possibility
- Hosting notebook servers on SURF Research Cloud? Perhaps hosting many individual Jupyter servers in Docker containers with the data preloaded in each? 
- Technically we should not use Google Collab due to the lack of a processing agreement with Google. 

For post-workshop distribution we can have a Zenodo repository for this GitHub repo, and another for the presentation and datasets.

**Comment by jelletreep** [comment]
I just found out about [solisworkspace](https://solisworkspace.uu.nl). It is a citrix cloud environment that all employees can use and it provides Jupyter Notebooks. I did a (very limited) test for our intro to python workshop and everything seems to work. The U drive is mounted in the enviroment. It is not as fluent as other solutions (e.g. short lag when creating files and folders), but it is easy to set up, everyone has access, and they are able to save notebooks and files into their own filesystem. If we choose this option we should test whether it is able to handle the size of the geospatial datafiles
